### PR TITLE
Law: exponential decay of damped oscillator

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

File: symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    print_expression,
+)
+
+# Description
+## For any (positive) value of the damping ratio, the position of a damped oscillator
+## is subject to exponential decay, i.e. its position is proportional to exp(-lambda*t)
+## where lambda is the exponential decay constant and t is time.
+
+# Law: lambda = omega * zeta
+## lambda - exponential decay constant
```
Comment (blackyblack) [inline symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py:19]: It is not really a constant. Maybe `decay rate` is better?

File: symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    validate_output,
+    print_expression,
+)
+
+# Description
+## For any (positive) value of the damping ratio, the position of a damped oscillator
+## is subject to exponential decay, i.e. its position is proportional to exp(-lambda*t)
+## where lambda is the exponential decay constant and t is time.
+
+# Law: lambda = omega * zeta
```
Comment (blackyblack) [inline symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py:18]: Maybe it is better to write it in form `zeta = lambda / omega`. I think it is more usual to know decay rate and calculate damping ratio from it.

Comment (blackyblack) [inline symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py:18]: It seems, it is approximate and for underdamped case.

Comment (alesanter) [reply symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py:18]: I'll look into it

File: symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py (hunk 1 above)
Comment (alesanter) [reply symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py:19]: image
It's not a constant, but the terminology seems to be like that

File: symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py (hunk 2 above)
Comment (alesanter) [reply symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py:18]: I guess you're right, it makes more sense practically

File: symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py (hunk 1 above)
Comment (blackyblack) [reply symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py]: ok

File: symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py (hunk 2 above)
Comment (blackyblack) [reply symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py]: What about this?

Comment (alesanter) [reply symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py]: Why do you think it is approximate? I don't fully understand what you mean by that word. The solution of the damped oscillator equation contain the term $`\exp(-\lambda t)`$, where $`\lambda = \zeta \omega_0`$ by design of that equation.

Comment (blackyblack) [reply symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py]: My mistake. I've seen someone mentioned it, but it seems I got it wrong.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**